packages/babel-types/src/definitions/experimental.js

```diff
@@ -61,23 +61,3 @@ defineType("ExportNamespaceSpecifier", {
     }
   }
 });
-
-defineType("RestProperty", {
```

Comment:
Should we add these as virtual types in https://github.com/babel/babel/blob/7.0/packages/babel-traverse/src/path/lib/virtual-types.js, e.g.

```
export const RestProperty = {
  types: ["RestElement"],
  checkPath(path: NodePath): boolean {
    return path.parentPath && path.parentPath.isObjectPattern();
  },
};
export const SpreadProperty = {
  types: ["RestElement"],
  checkPath(path: NodePath): boolean {
    return path.parentPath && path.parentPath.isObjectExpression();
  },
};
```

it might make it easier for people to convert a plugin from 6.x to 7.x